tests/agents/test_simple_chat_agent.py

```diff
@@ -9,6 +9,7 @@
 from neuroagent.agents import AgentOutput, AgentStep, SimpleChatAgent
 
 
+@pytest.mark.httpx_mock(can_send_already_matched_responses=True)
```

Comment:
just wondering. is this for the cases where we have sent the exact request before somewhere else in the tests?